filters/tracing/tagonresponse.go

```diff
@@ -0,0 +1,59 @@
+package tracing
+
+import (
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+)
+
+type tagOnResponseSpec struct {
+}
+
+type tagOnResponseFilter struct {
+	tagName  string
+	tagValue *eskip.Template
+}
+
+// NewTagOnResponse creates a filter specification for the tracingTagOnResponse filter.
+func NewTagOnResponse() filters.Spec {
+	return tagOnResponseSpec{}
+}
+
+func (s tagOnResponseSpec) Name() string {
+	return filters.TracingTagOnResponseName
+}
+
+func (s tagOnResponseSpec) CreateFilter(args []interface{}) (filters.Filter, error) {
+	if len(args) != 2 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	tagName, ok := args[0].(string)
+	if !ok {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	tagValue, ok := args[1].(string)
+	if !ok {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	return tagOnResponseFilter{
+		tagName:  tagName,
+		tagValue: eskip.NewTemplate(tagValue),
+	}, nil
+}
+
+func (f tagOnResponseFilter) Request(filters.FilterContext) {}
```

Comment:
let's use ptr receiver

Reply:
Done. This is my first piece of go code, so I hope I did it correctly. I checked other filters and copied from there.